src/ompl/util/RandomNumbers.h

```diff
@@ -152,6 +186,81 @@ namespace ompl
 
     };
 
+
```

Comment:
This is purely a helper class for prolate hyperspheroid (PHS) sampling, but it seems like a lot to place in RandomNumbers.h, should I move it to:
  src/ompl/util/ProlateHyperspheroid.h
?